typescript/web/bundlewatch.config.js

```diff
@@ -4,11 +4,11 @@ const bundlewatchConfig = {
     // Next JS client files
     {
       path: "./.next/static/chunks/**/*-*.js",
-      maxSize: "300kb",
+      maxSize: "350kb",
```

Comment:
Well, it'd need further investigation but that's possible since it adds a few KB into the bundle: 
![image](https://user-images.githubusercontent.com/2091902/152798147-fd603549-01e1-4cb7-8704-8154b2cedf3d.png)

I've had the same issue when adding Monaco on #828. And on #845, which is based on main, we can see that the total bundle is already at 361.62KB so it looks normal. (_it's more than 300 because in includes more than just `./.next/static/chunks/**/*-*.js` which has the `300kb` limit we're talking about_)

Reply:
react-query (which I've added too and will be used in other PRs) is also not innocent here: 
![image](https://user-images.githubusercontent.com/2091902/152803494-87d0ab08-9f6c-486b-b421-ed8a1ec22b61.png)